git_machete/docs.py

```diff
@@ -31,6 +32,8 @@
     "version": "Display the version and exit"
 }
 
+help_topics: List[str] = ['config']
```

Comment:
Hmmm can it be defined in tests instead? as it's not used anywhere in production code